### PR TITLE
Added the dodolist file for review.

### PR DESCRIPTION
**Pull request by addisalemseifu**
-Copy all the documents from the todo list file.

### CODE REVIEW

**Comment by Amazinggracee** [comment]
> -Copy all the documents from the to-do list file.

Hey, Addisalemseifu, It  would be nice if you make your PR more descriptive, for example

- [ ]  I gave feedback on someone else's project based on clean code best practices.

- [ ]  I applied the knowledge gained in the past when giving a code review.

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                <div class="my_head inputs">
+                    <h1 class="header_one ">Today's To Do</h1>
+                <i class="fa-solid fa-rotate"></i>
+                </div>
+                
+            <div>
+            <div class="inputs">
+                <input type="text" placeholder="Add to your list..." id="txtTitle" class="txtTitle" required>
+                <div class="addBook"><i class="fa-solid fa-arrow-left"></i></i></div>
+            </div>
```
Comment (Amazinggracee) [inline src/index.html:18]: Nice work dear, kindly close-up the blank lines, spaces, or indentations unless it is to separate large or logical code blocks.

File: src/index.css [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+}
+
+.trash_active {
+  cursor: pointer;
+  transform: translateY(0);
+  margin-right: 0.5rem;
+}
+
+.fa-ellipsis-vertical {
+  cursor: pointer;
```
Comment (Amazinggracee) [inline src/index.css:81]: To keep your code simple, kindly avoid duplicated code. Please check your lines 82, 58, etc.

**Review by Amazinggracee — CHANGES_REQUESTED** [review]
### Highlights
### **STATUS:  :recycle:**

196425615-8927e377-c543-407b-8da4-5c09fecc261b

- [ ] All linter checks are passing ✔️ ✔️
- [ ] Your project works as expected and is interactive 👏 👏
### Your project
Optional suggestions
OPTIONAL- After clearing all completed task, the index for the tasks do not reset, please fix this 🙏
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.css (hunk 2 above)
Comment (addisalemseifu) [reply src/index.css:81]: Thanks for the comment. I will fix that right away.

**Review by Amazinggracee — APPROVED** [review]
### Approved
unnamed
Nothing more to change, you can merge branch.

**Comment by addisalemseifu** [comment]
hey i am waiting for you.

**Pull request merged**